std/experimental/allocator/mmap_allocator.d

```diff
@@ -72,10 +74,10 @@ struct MmapAllocator
     }
 }
 
-@system unittest
+nothrow @safe @nogc unittest
 {
     alias alloc = MmapAllocator.instance;
     auto p = alloc.allocate(100);
     assert(p.length == 100);
-    () nothrow @nogc { alloc.deallocate(p); }();
+    () @trusted { alloc.deallocate(p); }();
```

Comment:
Also need to nullify `p` in the `@trusted` section. Else you have a dangling pointer in `@safe` code.